tests/integration/test_events.py

```diff
@@ -27,79 +20,95 @@ def setUp(self):
         self.events_client = aws_stack.connect_to_service('events')
 
     def test_put_rule(self):
-        self.events_client.put_rule(Name=TEST_RULE_NAME, EventPattern=json.dumps(TEST_EVENT_PATTERN))
-        rules = self.events_client.list_rules(NamePrefix=TEST_RULE_NAME)['Rules']
+        rule_name = 'rule-{}'.format(short_uid())
+
+        self.events_client.put_rule(
+            Name=rule_name,
+            EventPattern=json.dumps(TEST_EVENT_PATTERN)
+        )
+
+        rules = self.events_client.list_rules(NamePrefix=rule_name)['Rules']
 
         self.assertEqual(1, len(rules))
         self.assertEqual(TEST_EVENT_PATTERN, json.loads(rules[0]['EventPattern']))
 
-    def test_put_event(self):
-        response = self.events_client.put_events(Entries=[{
-            'Time': datetime(2019, 7, 29),
-            'DetailType': TEST_DETAIL_TYPE,
-            'Detail': TEST_DETAIL
-        }])
-        entries = response['Entries']
-        self.assertEqual(1, len(entries))
-        event_id = entries[0]['EventId']
-        self.assertRegex(event_id, '[0-9a-f-]{36}')
-
-    def test_events_written_to_disk_are_timestamp_prefixed_for_chronological_ordering(self):
-        event_type = str(uuid.uuid4())
-        event_details_to_publish = list(map(lambda n: 'event %s' % n, range(100)))
-
-        for detail in event_details_to_publish:
-            self.events_client.put_events(Entries=[{
-                'DetailType': event_type,
-                'Detail': detail
-            }])
-
-        sorted_events_written_to_disk = map(
-            lambda filename: json.loads(str(load_file(os.path.join(EVENTS_TMP_DIR, filename)))),
-            sorted(os.listdir(EVENTS_TMP_DIR)))
-        sorted_events = list(filter(lambda event: event['DetailType'] == event_type,
-                                    sorted_events_written_to_disk))
-        self.assertListEqual(event_details_to_publish, list(map(lambda event: event['Detail'], sorted_events)))
+        # clean up
+        self.events_client.delete_rule(
+            Name=rule_name,
+            Force=True
+        )
+
+    # TODO Can be removed?
```

Comment:
I think we should still leave this functionality (and the tests), as some users are relying on events being written to disk. In the future, we may want to add a configuration flag to enable/disable this feature.